examples/lab_devices/keithley6517a_pyvisa.yaml

```diff
@@ -0,0 +1,22 @@
+#################################################
+# An example for setting up the remote connection
+# Last Modified: Mi 01 Jun 2022 07:31:34  CEST
+#################################################
+
+transfer_layer:
+  - name: electrometer
+    type: Visa
+    init:
+        # using serial-USB adapter
+        resource_name: ASRLCOM6         # example for windows system
+#        resource_name: ASRL/dev/ttyUSB0 # example for linux system 
+        timeout : 1000
+        read_termination: "\n"
+        baud_rate: 19200
```

Comment:
Is 19200 the standard baud rate of the device when powered on or did you set this?

Reply:
is the default. the read termination cannot be set, so the value there is based on experience

Reply:
Is the `timeout` in ms?

Reply:
Can you add a `query_delay: 1.0` to the transfer layer `init`? This should do the sleeping you are putting manually